src/RemoteConfigContext.tsx

```diff
@@ -1,10 +1,14 @@
 import React, {createContext, useContext, useEffect, useState} from 'react';
 import remoteConfig from '@react-native-firebase/remote-config';
-import {RemoteConfig, defaultRemoteConfig, getConfig} from './remote-config';
+import {defaultRemoteConfig, getConfig, RemoteConfig} from './remote-config';
 import Bugsnag from '@bugsnag/react-native';
 
 import {useAppState} from './AppContext';
 import {FeedbackConfiguration} from '@atb/components/feedback';
+import useInterval from '@atb/utils/use-interval';
+
+/** The retry interval when the Remote Config fetch failed */
+const RETRY_INTERVAL_MS = 15000;
```

Comment:
Is 15s a bit much here? I guess we have to account for slow connections at times, but think I would expect the app to retry more often if I lost reception. (And better than nothing for sure, so just asking 😄)

Reply:
Triggering it more often might be better for the user, but it is risky if we actually have some errors in our code, and end up triggering all the time on multiple devices. But I think I will invest some time into doing a quicker first retry, with exponential backoff so that it doesn't continuously trigger as often.

Reply:
Added now. Did this locally in RemoteConfigContext for now, and we can extract it later if we want to use it more places.